docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -16,7 +16,7 @@ Update the `production` settings with the IP or domain name where the project wi
 
 ```js
 {
-  "host": "domain.io", // IP or domain or 0.0.0.0
+  "host": "domain.io",// IP or domain or 0.0.0.0
```

Comment:
domain should be removed here, that isn't something that should be directly entered as it requires the OS to do DNS lookups and if the DNS fails the server will fail to bind a NaN IP address (typically gathered from reverse lookups) it also opens a vector of attack with DNS poisoning. In theory using a domain here -should- work, but it's nothing something we should recommend.

Previously it didn't matter because Strapi wasn't actually binding to anything here, but as of beta.19.4/19.5 it does matter.

Reply:
Yes I agree this should be replaced with a combination of host + proxy (soon url)